packages/shared/src/write-archive/index.ts

```diff
@@ -14,6 +14,9 @@ import { createStories, storiesFileName } from './stories-files';
 // archive/<test-title>.json
 // archive/<file>.<ext>
 
+// const ARCHIVE_DENYLIST: string[] = ['http://localhost:6006/index.json'];
+const ARCHIVE_DENYLIST: RegExp[] = [/'^(https?:\/\/)localhost:(\d+)\/index\.json'/i];
```

Comment:
How do we know that we shouldn't archive this for all projects?

Reply:
And why does this need to be denied for Storybook projects? What problem is it causing?

Reply:
We don't. Yet, if that file exists in the archive, the archive storybook will use it to power the Storybook and cause issues.

Reply:
Specifically, if the `index.json` file exists in the build archive, when the build-archive-storybook generates the Storybook to send to Chromatic, the archived `index.json` overwrites the generated `index.json` from building the Storybook, which ultimately causes significant problems rendering the SB properly because the rendering is driven by this file.

The archived index.json includes a list of stories from the storybook that we ran e2e on.
The generated index.json includes a list of our archives that we generated to render.

We ultimately want the generated one to stick around and do not care at all about the archived one because we have a DOM Snapshot.